src/framework/core/js/FluidDocument.js

```diff
@@ -174,4 +174,17 @@ var fluid_2_0_0 = fluid_2_0_0 || {};
         };
     });
 
+    /* Sets the value to the DOM element and triggers the change event on the element.
+     * Note: when using jQuery val() function to change the node value, the change event would
+     * not be fired automatically, it requires to be initiated by the user.
+     *
+     * @param node {A jQueryable DOM element} A selector, a DOM node, or a jQuery instance
+     * @param value {String|Number|Array} A string of text, a number, or an array of strings
+     * corresponding to the value of each matched element to set in the node
+     */
+    fluid.changeElementValue = function (node, value) {
```

Comment:
Thanks - these docs look good now.
In checking over the diff, I see that there are still actually no calls to this utility outside test cases - which suggests that the proper place for it is inside our test utilities rather than the core framework. 

I suggest it should go into the file jqUnit-browser.js under the name jqUnit.changeElementValue